### PR TITLE
Release notes for 2020.3.3

### PR DESCRIPTION
**Pull request by digisomni**
This PR also tags along a couple of reference updates around the repo as well. I will want to update the logo to use a more clear "tm" picture however. I intend to probably use the SVG logo and create variants from that so it's consistent.

### CODE REVIEW

**Review by JulianGro — CHANGES_REQUESTED** [review]
Could you downscale that logo? We are never even going bigger than 500 pixel width when scaling the page. Maybe go ~1000 horizontal pixels in case someone uses 2x DPI?
This just wastes resources.

File: docs/source/host/add-content/bake-content.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ Most content (avatars, entities, etc) in High Fidelity references external resou
 The Oven
 ---------------------
 
-The Oven is a standalone application that is packaged as part of High Fidelity `Client + Sandbox <https://www.highfidelity.com/download#sandbox>`_ installer. Once installed, locate ``oven.exe`` (Windows) or ``oven.app`` (Mac).
+The Oven is a standalone application that is packaged as part of High Fidelity `Client + Sandbox <https://vircadia.com/deploy-a-server/>`_ installer. Once installed, locate ``oven.exe`` (Windows) or ``oven.app`` (Mac).
```
Comment (ctrlaltdavid) [inline docs/source/host/add-content/bake-content.rst:14]:
```suggestion
The Oven is a standalone application that is packaged as part of Vircadia `Client + Sandbox <https://vircadia.com/deploy-a-server/>`_ installer. Once installed, locate ``oven.exe`` (Windows) or ``oven.app`` (Mac).
```

File: docs/source/release-notes/2020-3-3-Demeter.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    * Main "Edit" menu:
+        * The two "Import Entities" actions have been brought back to the main "Edit" menu. (Since there was no unanimity about their removal.)
+        * A separator has been added to properly group the 'Create' app's menu items.
+    * In desktop mode only, the default window width is now 750px instead of 490px.
+* App: 'Create' app material icons update. (#907)
+    * Preference: "Show Zones in Create Mode" has been removed. The method behind it is still in the API, but does nothing. It has been abandoned since it was only drawing a box that wasn't useful to assist in figuring out the shape of the zone.
+    * Preference: "Show Lights and Particle Systems in Create Mode" has been renamed to "Show Icons in Create Mode" since this setting was also managing all the icons (including Zones and now Parented Materials). The objective of the setting is mainly to simplify the display in edit mode when a scene is very dense.
+    * `image3D` overlays used for the icons have been migrated to Local Entities.
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2020-3-3-Demeter.md:41]:
```suggestion
    * `image3D` overlays used for the icons have been migrated to local entities.
```

File: docs/source/release-notes/2020-3-3-Demeter.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    * All the Entities icons (in-world non physical entities) has been updated to be more uniform. They now all have the same dark gray tone, with a white border to keep them visible under different lighting.
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2020-3-3-Demeter.md:42]:
```suggestion
    * All the entity icons (in-world non physical entities) have been updated to be more uniform. They now all have the same dark gray tone, with a white border to keep them visible under different lighting.
```

File: docs/source/release-notes/2020-3-3-Demeter.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+### Iamus [Metaverse Server](https://github.com/vircadia/iamus)
+
+#### Version 2.3.6
+
+<ul>
+<li><a href="http://github.com/vircadia/Iamus/commit/d39b5731e0481cc0006dae2e6fb4a8530137b0bb">view &bull;</a> Update ChangeLog.md</li> 
+<li><a href="http://github.com/vircadia/Iamus/commit/c67706945d71c51d806023c02dce26d4147ea1a6">view &bull;</a> Minor bug fixes in BackupDb.sh</li> 
+<li><a href="http://github.com/vircadia/Iamus/commit/6b05304c326f8303a18afbde9421ccf030949297">view &bull;</a> Update links and names (#66)</li> 
+<li><a href="http://github.com/vircadia/Iamus/commit/cc8e9d64f077c2ee62405cfe2437ad3483845c24">view &bull;</a> Update NPM package version.</li> 
+<li><a href="http://github.com/vircadia/Iamus/commit/8cf1eee04aab3a069442c346241cf96aa8572f1a">view &bull;</a> Fix problem where user wasn't returned their token by /api/v1/tokens     The search field must be specified to AccountScopeFilter     A few other places were checked and updated.</li>
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2020-3-3-Demeter.md:72]:
```suggestion
<li><a href="http://github.com/vircadia/Iamus/commit/8cf1eee04aab3a069442c346241cf96aa8572f1a">view &bull;</a> Fix problem where user wasn't returned their token by /api/v1/tokens. The search field must be specified to AccountScopeFilter. A few other places were checked and updated.</li> 
```

File: docs/source/release-notes/2020-3-3-Demeter.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+<li><a href="http://github.com/vircadia/Iamus/commit/cc8e9d64f077c2ee62405cfe2437ad3483845c24">view &bull;</a> Update NPM package version.</li> 
+<li><a href="http://github.com/vircadia/Iamus/commit/8cf1eee04aab3a069442c346241cf96aa8572f1a">view &bull;</a> Fix problem where user wasn't returned their token by /api/v1/tokens     The search field must be specified to AccountScopeFilter     A few other places were checked and updated.</li> 
+<li><a href="http://github.com/vircadia/Iamus/commit/1d0be3bf637315c3196390124600cada3fc218f1">view &bull;</a> Bump version to 2.3.6</li> 
+</ul>
+
+#### Version 2.3.5
+
+<ul>
+<li><a href="http://github.com/vircadia/Iamus/commit/e95291d4988f48d780795e134a364d9120f22fc3">view &bull;</a> Update ChangeLog.md</li> 
+<li><a href="http://github.com/vircadia/Iamus/commit/b360944a70a994ee701a0f2b23303a1d71dd76fb">view &bull;</a> Change Place filter 'maturity' to take a comma separated string     of maturity classes to match.</li>
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2020-3-3-Demeter.md:80]:
```suggestion
<li><a href="http://github.com/vircadia/Iamus/commit/b360944a70a994ee701a0f2b23303a1d71dd76fb">view &bull;</a> Change Place filter 'maturity' to take a comma separated string of maturity classes to match.</li> 
```

File: docs/source/release-notes/2020-3-3-Demeter.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+</ul>
+
+#### Version 2.3.5
+
+<ul>
+<li><a href="http://github.com/vircadia/Iamus/commit/e95291d4988f48d780795e134a364d9120f22fc3">view &bull;</a> Update ChangeLog.md</li> 
+<li><a href="http://github.com/vircadia/Iamus/commit/b360944a70a994ee701a0f2b23303a1d71dd76fb">view &bull;</a> Change Place filter 'maturity' to take a comma separated string     of maturity classes to match.</li> 
+<li><a href="http://github.com/vircadia/Iamus/commit/00355c7481b1958379d574a1ff7acc65a18d2eed">view &bull;</a> When creating a Place, default maturity to the parent domain's maturity. Default DomainEntity.maturity to Maturity.UNRATED.</li> 
+<li><a href="http://github.com/vircadia/Iamus/commit/a80864dfea02c284c195000184392f9c1f7bdcce">view &bull;</a> Massive refactoring to remove circular import dependencies that were     causing null value initializations due to how Javascript loads. Move Entity field definitions into separate files. Moved Entity field getting and setting into the Entity controlling class. Repackaged (into multiple files) Permission and Get/Set/Validate routines.</li>
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2020-3-3-Demeter.md:82]:
```suggestion
<li><a href="http://github.com/vircadia/Iamus/commit/a80864dfea02c284c195000184392f9c1f7bdcce">view &bull;</a> Massive refactoring to remove circular import dependencies that were causing null value initializations due to how JavaScript loads. Move Entity field definitions into separate files. Moved Entity field getting and setting into the Entity controlling class. Repackaged (into multiple files) Permission and Get/Set/Validate routines.</li> 
```

File: docs/source/release-notes/2020-3-3-Demeter.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+<li><a href="http://github.com/vircadia/Iamus/commit/acaa36a494ad439122f48376ba31f08068219566">view &bull;</a> Added "maturity" and "tags" to PlaceEntity Centralized "maturity" specifications into src/Entities/sets/Maturity.ts Moved AccountAvailability and AccountRoles to src/Entities/sets and made a set pattern. Documented "/api/v1/places?maturity=LEVEL" and "/api/v1/places?tags=TAGLIST" Added comments to src/Entities/EntityFilters/CriteriaFilter.ts explaining how filters are to be used Added "maturity-catagories" to Places responses Fixed /api/v1/users/places to return only the places that belong to the requesting user Deleted bin/www -- what was that from??</li> 
+<li><a href="http://github.com/vircadia/Iamus/commit/56c4649b60f58915c3b05907f3f1149cf3fe0e71">view &bull;</a> Bump version to 2.3.4</li> 
+</ul>
+
+### Iamus [Metaverse Dashboard](https://github.com/vircadia/project-iamus-dashboard)
+
+Updates are reflected [here](https://dashboard.vircadia.com/). The Dashboard has been upgraded from version 1.1.3 to version 1.2.2. (13 commits altering functionality, 18 commits total)
```
Comment (ctrlaltdavid) [inline docs/source/release-notes/2020-3-3-Demeter.md:100]:
```suggestion
Updates are reflected [here](https://dashboard.vircadia.com/). The Dashboard has been upgraded from version 1.1.3 to version 1.2.2. (13 commits altering functionality, 18 commits total.)
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

File: docs/source/create/entities/display-youtube.md [hunk 8]
```diff
@@ -1,12 +1,12 @@
...
 
-In this tutorial, you will learn how web entities work by creating one displaying a YouTube channel. You can watch videos in High Fidelity using this web entity. 
+In this tutorial, you will learn how web entities work by creating one displaying a YouTube channel. You can watch videos in Vircadia using this web entity. 
 
 **On This Page:**
 
 - [Prerequisites](#prerequisities)
 - [Create a Web Entity](#create-a-web-entity)
-- [Display High Fidelity's YouTube Channel](#display-high-fidelitys-youtube-channel)
+- [Display Vircadia's YouTube Channel](#display-high-fidelitys-youtube-channel)
```
Comment (JulianGro) [inline docs/source/create/entities/display-youtube.md:9]: Link is broken
```suggestion
- [Display Vircadia's YouTube Channel](#display-vircadia-s-youtube-channel)
```

File: docs/source/home.rst [hunk 9]
```diff
@@ -2,7 +2,7 @@
 Home
 ####
 
-Vircadia is open-source software which enables you to create and share virtual worlds as virtual reality (VR) and desktop experiences. You can create and host your own virtual world,
+Vircadia™ is open-source software which enables you to create and share virtual worlds as virtual reality (VR) and desktop experiences. You can create and host your own virtual world,
```
Comment (JulianGro) [inline docs/source/home.rst:5]: This seems to be up for debate. The (TM) is supposedly not required by law and only makes it sound like we are going to be asshats about people using the name.

File: docs/source/release-notes/2020-3-3-Demeter.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
+# Alpha 2020.3.3 Demeter Release Notes
+
+This version of Vircadia is an update to 2020.3.2. It is compatible with the 2020.3.0-Demeter protocol. This version was released on December 30th, 2020.
```
Comment (JulianGro) [inline docs/source/release-notes/2020-3-3-Demeter.md:3]:
```suggestion
This version of Vircadia is an update to 2020.3.2. It is compatible with the 2020.3.0-Demeter protocol. This version was released on January 2th, 2021.
```

File: docs/source/release-notes/2020-3-3-Demeter.md [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+* Server: The Domain server now sends its IP address and port to the Metaverse server with every heartbeat. (#877)
+
+## Metaverse Server & Dashboard (Codename Iamus)
+
+### Iamus [Metaverse Server](https://github.com/vircadia/iamus)
+
+#### Version 2.3.6
+
+<ul>
+<li><a href="http://github.com/vircadia/Iamus/commit/d39b5731e0481cc0006dae2e6fb4a8530137b0bb">view &bull;</a> Update ChangeLog.md</li>
```
Comment (JulianGro) [inline docs/source/release-notes/2020-3-3-Demeter.md:68]: Why do these all have double bullet points?

File: docs/source/release-notes.rst [hunk 12]
```diff
@@ -8,6 +8,7 @@ See what's been released in Vircadia.
     :maxdepth: 2
     :titlesonly:
 
+    Alpha Release 2020.3.3 Demeter (2020-12-30) <release-notes/2020-3-3-Demeter>
```
Comment (JulianGro) [inline docs/source/release-notes.rst:11]:
```suggestion
    Alpha Release 2020.3.3 Demeter (2021-01-02) <release-notes/2020-3-3-Demeter>
```

**Review by JulianGro — CHANGES_REQUESTED** [review]

File: docs/source/home.rst (hunk 9 above)
Comment (digisomni) [reply docs/source/home.rst:5]: It's mainly to be safe. I have to grant permission to use the name in order to protect the trademark for the project. Giving notice helps with confusion as we've also seen as well. lol

File: docs/source/release-notes/2020-3-3-Demeter.md (hunk 11 above)
Comment (digisomni) [reply docs/source/release-notes/2020-3-3-Demeter.md:68]: They are generated using a script in the Iamus repo I believe. Just a way to format it I suppose, I copy and paste it over from here https://github.com/vircadia/Iamus/blob/master/ChangeLog.md.

**Review by JulianGro — APPROVED** [review]
Good enough for release notes.

**Pull request merged**